modules/linear_boltzmann_solvers/lbs_solver/iterative_methods/ags_linear_solver.h

```diff
@@ -12,55 +12,38 @@ namespace lbs
 {
 
 /**Linear Solver specialization for Within GroupSet (WGS) solves.*/
-class AGSLinearSolver : public LinearSolver
+class AGSLinearSolver
```

Comment:
Unless there is a notion of a nonlinear AGS solver, I would probably go with `AGSSolver`.